exporter/metric/metric.go

```diff
@@ -272,25 +291,92 @@ func (me *metricExporter) recordToMdpb(record *export.Record) *googlemetricpb.Me
 	}
 }
 
+// checkPrefixInMapKeys checks whether the non-empty prefix string
+//  is in the values of the map
+func checkPrefixInMapKeys(labelMap map[string]string, prefix string) bool {
+	if len(prefix) == 0 {
+		return false
+	}
```

Comment:
^ I'd probably get rid of that condition. I don't think you have any tests that cover that condition, and technically an empty string is a prefix of any key so this should return true anyway.